File: PATTERNS/Pattern03/Pattern3.java
//*
//**
//***
//****
//*****
//****
//***
//**
//*
import java.util.Scanner;
public class Pattern3{
    public static void main(String[] args) {
        Scanner in  = new Scanner(System.in);
        System.out.println("Enter the row number:");
        int n = in.nextInt();

        int j = 1, k = 1,i=1;
        for(i=1;i<=(2*n-1);i++){
           k = (i<=n)? i:(2*n-i);
           for(j=1;j<=k;j++){
               System.out.print("*");
           }
            System.out.println();
        }
    }
}

File: PATTERNS/Pattern03/Pattern3.py

# *
# **
# ***
# ****
# *****
# ****
# ***
# **
# *

n = int(input("enter the number of rows:"))

j = 1
k = 1
i = 1
for i in range(2 * n):
    k = i if(i <= n) else (2 * n - i)
    for j in range(1,k+1):
        print("*",end='')
    print()